README.md

```diff
@@ -214,6 +214,17 @@ A number of options are available to allow you to customise the SDK:
 
   A string of the ID of the container element that the UI will mount to. This needs to be an empty element. The default ID is `onfido-mount`.
 
+- **`language {Object} optional`**
+  An object that includes the selected locale. If `language` is not present the locale will default to `en`.
+  In order to override the default translation of a string it is possible to use the option `customStrings` that accepts an object containing the key and the new value. The keys are available with `./locales/en.js`
+
+    ```javascript
+    language: {
+      locale: 'es', // required
+      customStrings: {some_key: 'This is my custom string'}
```

Comment:
I don't think we should allow runtime customisation of strings. It adds complexity and I don't imagine anyone will actually use it.

@rfreitas @seewah thoughts?

Reply:
I agree, I would remove it for now.